invenio_records_resources/records/systemfields/files/manager.py

```diff
@@ -325,9 +342,6 @@ def __getitem__(self, key):
         value = self.entries.get(key)
         if isinstance(value, self.file_cls):
             return value
-        # TODO: implement "file_cls.loads/from_dict"
```

Comment:
Not needed, the ``FilesField`` is now responsible for injecting proper file records instead of loading them here,.